doc/src/start/builds/minimal.md

```diff
@@ -31,7 +31,7 @@ Opt-Out of Features
 ------------------
 
 Opt out of as many features as possible, if they are not needed, to reduce code size because, remember, by default
-all code is compiled into the final binary since what a script requires cannot be predicted.
+all code except [`decimal`] is compiled into the final binary since what a script requires cannot be predicted.
```

Comment:
Well, actually there are a few opt-in features, such as `serde`.  So this statement is not technically correct.